### ISSUE TITLE
4.0.3 installation failed

### ISSUE THREAD

**Issue by Giuse69** [Octopoos/SEBLOD#718]
Hi, when trying to update Seblod 3.24 to 4.0.3 on Joomal 3.10.11 (latest) with latest Updater v1.5.0, I get this error
`Unknown column 'cck' in 'jtab_cck_store_item_content'
Package Update: There was an error installing an extension: plg_content_cck.zip`
and the installation throws a failure message.
Anything to check?
thanks

Giuse

**Comment by NativoDigital** [comment]
Hi @Giuse69,
If you site runs on Joomla! 3 and you'd like to stay on Joomla! 3 you should remain on SEBLOD 3

[Source](https://www.seblod.com/community/blog/seblod-4-0-series-4-4)

**Comment by Giuse69** [comment]
Hi, no, I want to upgrade to J4, so I followed the suggested path first upgrading Seblod to 4 but getting the error..
Cheers

**Comment by sebastienheraud** [comment]
Hi @Giuse69,

OK we were able to replicate the upgrade issue, and the thing is the following:
The SEBLOD 4 package (depending on which one and when you try to update) may not be aware of the SEBLOD 3 release... it's trying to update. And it that case the SQL queries update fails.
ex:
* SEBLOD 3.24 can be upgraded with SEBLOD 4.0.5 without any issue
* SEBLOD 3.25 is released and trying to update an existing 3.25 site with SEBLOD 4.0.5 will fail.
* the existing 3.25 site will require the 4.0.6 package so the upgrade is completed properly

So we need to release a new 4.x package after each 3.x package, and you (when upgrading a 3.x site) need to make sure you have the latest 4.x available (if it was updated to the latest SEBLOD 3.x)

On a site where the upgrade failed the 1st time, you just need to manually run ALL queries found in /administrator/components/com_cck/install/upgrades/4.x.x.sql (each 4.x.x file) via PhpMyAdmin to have the SQL schema up-to-date.

Saba
